Gemfile

```diff
@@ -0,0 +1,2 @@
+source 'https://rubygems.org'
+gem 'github-pages', group: :jekyll_plugins
```

Comment:
I suggest adding a few extra lines here: 
- `frozen_string_literal` -- make strings unchangeable. Usual practice in Ruby [1](https://stackoverflow.com/a/37799399/7410631).
- `git_source`: workaround for a bug in Bundler [2](https://stackoverflow.com/a/41454572/7410631)
- `ruby '>=2.5.3'` -- to make sure people use an up-to-date Ruby used by GitHub

```
# frozen_string_literal: true

source "https://rubygems.org"
git_source(:github) {|repo_name| "https://github.com/#{repo_name}" }

# Synchronize with https://pages.github.com/versions
ruby '>=2.5.3'

gem 'github-pages', group: :jekyll_plugins
```